components/google-cloud/google_cloud_pipeline_components/experimental/forecasting/prepare_data_for_train/component.yaml

```diff
@@ -0,0 +1,249 @@
+name: Prepare data for train
+description: Prepares the parameters for the training step.
+inputs:
+- name: input_tables
+  type: String
+  description: |-
```

Comment:
I don't know if the readthedoc can generate the documentation successfully when you put the description inline with every parameter. 
You can merge this and wait for the next release to give a try. here is the doc site
https://readthedocs.org/projects/google-cloud-pipeline-components/